CIP-0100/action-example.jsonld

```diff
@@ -0,0 +1,35 @@
+{
+  "@context": {
+    "paymentAddress": "https://github.com/cardano-foundation/CIPs/blob/master/CIP-0100/common-properties.jsonld#payment-address",
+    "displayName": "https://github.com/cardano-foundation/CIPs/blob/master/CIP-0100/common-properties.jsonld#displayName",
+    "DID": "https://github.com/cardano-foundation/CIPs/blob/master/CIP-0100/common-properties.jsonld#DID",
+    "language": "https://datahub.io/core/language-codes/r/0.html",
```

Comment:
It looks like json-ld has native understanding of BCP47 language codes: https://www.w3.org/ns/json-ld#language

It also has support for providing multiple translations for *any* field:

https://www.w3.org/TR/json-ld11/#language-indexing

I may update the draft with these in mind.